AFMReader/jpk.py

```diff
@@ -35,7 +35,62 @@ def _jpk_pixel_to_nm_scaling(tiff_page: tifffile.tifffile.TiffPage) -> float:
     return px_to_nm * 1e9
 
 
-# pylint: disable=too-many-locals
+def _get_z_scaling(tif: tifffile.tifffile, channel_idx: int) -> tuple[float, float]:
+    """
+    Extract the z scaling factor and offset for a JPK image channel.
+
+    Determined using JPKImageSpec.txt Version: 2.0:3fffffffffff supplied with JPK instrument.
+
+    Parameters
+    ----------
+    tif : tifffile.tifffile
+        A tiff file of .jpk images.
+
+    Returns
+    -------
+    tuple
+        A tuple contains values used to scale and offset raw data.
+    """
+    NrOfSlots = tif.pages[channel_idx].tags["32896"].value
```

Comment:
As per [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names) we use `snake_case` rather than `CamelCase` across TopoStats and AFMReader for variable names.

If you could replace `NrOfSlots` with `n_slots` that would be great, it uses a common convention (`n`) to indicate its a count of something.

```suggestion
    n_slots = tif.pages[channel_idx].tags["32896"].value
```